### ISSUE TITLE
Startreihenfolge 2. Durchgang

### ISSUE THREAD

**Issue by MUELTHOM3** [RaceHorology/RaceHorology#613]
**Ist der Featurewunsch relevant für ein bestimmtes Problem? Bitte beschreibe dies.**
In kleineren Rennserien z.B. Ziener Cup starten die Athleten im nächsten Durchgang in umgekehrter Startreihenfolge (pro Klasse oder Gruppe). Jedoch soll für die Berechnung der Startliste des nächsten Durchgangs (z.B. 2. Durchgang) die ausgeschiedenen Läufer aus dem vorherigen Durchgang (z.B. 1. Durchgang) mit berücksichtigt werden. Aktuell wir "nur" Startnummer (absteigend) angeboten, doch berücksichtigt diese Selektion NICHT, wer im Durchgang zuvor ausgeschieden (NAS, DIS, NIZ, etc.) war.

**Beschreibe die Lösung die Du gerne hättest**
Neue Selektion Option: Vorheriger Lauf - Startnummer (absteigend) ==> Bei dieser Selektion werden je gewählter Gruppierung die aus dem vorherigen Durchgang ausgeschiedenen Athleten (NAS, DIS, NIZ, etc.) bei der Berechnung der Startliste nicht berücksichtigt.

**Comment by svenflossmann** [comment]
@MUELTHOM3 Ja, macht Sinn, diese Option anzubieten. Baue ich ein, sobald es reinpasst.

**Comment by BerndKlostermeyer** [comment]
@svenflossmann und vergiss bitte #253 nicht. Im Grunde ist das für alle Rennen gleich, egal ob 2, 3 oder mehr Durchgänge: Nicht vorheriger Lauf nach Zeit wie aktuell sondern Summe aller Läufe nach Zeit. Thx Bernd ;)
